src/dmn/trn_rpc_protocol_handlers_1.c

```diff
@@ -284,6 +284,49 @@ int *update_ep_1_svc(rpc_trn_endpoint_t *ep, struct svc_req *rqstp)
 	return &result;
 }
 
+int *update_packet_metadata_1_svc(rpc_trn_packet_metadata_t *packet_metadata, struct svc_req *rqstp)
+{
+	UNUSED(rqstp);
+	static int result = -1;
+	result = 0;
+	int rc;
+	char *itf = packet_metadata->interface;
+	struct packet_metadata_key_t key;
+	struct packet_metadata_t value;
+
+	TRN_LOG_DEBUG("update_packet_metadata_1 packet_metadata tunid: %ld, ip: 0x%x,"
+		      " pod_label_value: %d, namespace_label_value: %d",
+		      packet_metadata->tunid, packet_metadata->ip, packet_metadata->pod_label_value, packet_metadata->namespace_label_value);
+
+	struct user_metadata_t *md = trn_itf_table_find(itf);
```

Comment:
you are using user_metadata_t here, but you are updating from agent side...

Reply:
The above function update_ep_1_svc is using same pattern.